todo_app/trello_items.py

```diff
@@ -0,0 +1,77 @@
+import os
+import requests
+import json
+
+# Get environment variables
+TRELLOAPIKEY = os.getenv('TRELLOAPIKEY')
+TRELLOAPITOKEN = os.environ.get('TRELLOAPITOKEN')
+TODOID='6048d81adf46ed3c0e396169'
+DOING='6048d81adf46ed3c0e39616a'
+def Get_items_from_trello():
+    print(TRELLOAPIKEY)
+    print(TRELLOAPITOKEN)
```

Comment:
Let's remove these print statements now you have this working - generally good practice to avoid littering messages out, but particularly here where we'd worry about leaking these secrets to log files